src/Traits/StaticFromJson.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Eppo\Traits;
+
+trait StaticFromJson
```

Comment:
Oh god multiple inheritance has entered the chat 😅 

But this is actually a good use case so I'll allow it